evalai/utils/common.py

```diff
@@ -133,34 +152,63 @@ def generate_random_string(length):
 def upload_file_using_presigned_url(challenge_phase_pk, file, file_type, submission_metadata={}):
     if file_type == "submission":
         url = "{}{}".format(get_host_url(), URLS.get_presigned_url_for_submission_file.value)
+        upload_complete_url = "{}{}".format(get_host_url(), URLS.finish_upload_for_submission_file.value)
```

Comment:
My bad, as I missed these in last review --
`upload_complete_url` -> `finish_upload_url`